### ISSUE TITLE
Error on empty calculator input

### ISSUE THREAD

**Issue by numel007** [Grxcelynx/Calculate-and-Create-HBProject2021#1]
No input on canvas size, weather, or paint type causes an insertion error, likely because the model is expecting an id but is receiving a string instead. 

I'm thinking you could set the value of empty inputs to `0` instead of having an option tag with no `value` attribute, as it is right now. And then check for the 0 value when inserting/calculating.

**Comment by Grxcelynx** [comment]
I noticed that as well, thank you!

**Comment by numel007** [comment]
Thankfully it seems to be a pretty quick fix. From what I can tell, the values of the options correspond to the id of each paint/canvas/weather object and the route locates the corresponding db object to calculate from. Perhaps modifying `seeding_database.py`to create a value for empty inputs would also be an option? Or add some error handling directly into the route? There's a few ways to handle it.

**Comment by Grxcelynx** [comment]
Good idea and thank you for your advice! Very helpful. This project is my sort of test case for the "real" Calculate and Create so I will be utilizing my errors from this to better develop the 2.0 haha.
